test/torchaudio_unittest/models/wav2vec2/model_test.py

```diff
@@ -33,6 +35,8 @@ def _name_func(testcase_func, i, param):
         (hubert_base,),
         (hubert_large,),
         (hubert_xlarge,),
+        (wavlm_base,),
+        (wavlm_large,),
```

Comment:
Looks like adding wavlm configs to existing one adds many skip/branch conditions to existing tests. (We should avoid introducing always-skip as much as possible.)

I think it's better to logically split them. Can you create a dedicated test suit for wavlm?